spec/system/prior_authority/view_applications_spec.rb

```diff
@@ -175,8 +175,11 @@
     end
 
     it 'lets me download my uploaded file' do
-      click_on 'evidence.pdf'
+      click_on 'evidence–with-weird-char.pdf'
       expect(page).to have_current_path(%r{/test_path})
+      expect(page.driver.request.params['response-content-disposition']).to eq(
+        'attachment; filename="evidencewith-weird-char.pdf"'
```

Comment:
it looks like its just removing the _first_ hyphen here - what is the weird char?

Reply:
@jsugarman annoyingly GitHub doesn't highlight it, but my IDE does - that first hyphen isn't a hyphen, it's an "en dash", which isn't included in ISO-8859-1 and therefore isn't accepted by S3.

<img width="572" alt="Screenshot 2024-07-12 at 11 57 48" src="https://github.com/user-attachments/assets/7a0308e0-faed-43ff-870a-e983c75a3d2d">

Reply:
I've added a comment to make clearer what's going on.